test/test_derivs.jl

```diff
@@ -8,38 +8,48 @@ using Transform
 import Synthetic
 import WCS
 
-import GSL.deriv_central
+import ForwardDiff
+
 
 println("Running derivative tests.")
 
+function do_deriv_test(f::Function, x::Float64, claimed_dx)
+    # TODO: really should call ForwardDiff.gradient just once, with a whole
+    # vector of x values, rather than once per parameter to test
+    fwd_deriv = ForwardDiff.gradient(f)([x,])[1]
+
+    info("got $fwd_deriv; expected $claimed_dx")
+
+    @test_approx_eq_eps significand(fwd_deriv) significand(claimed_dx) 1e-4
```

Comment:
Although all this is actually going to be replaced soon by the tests in the ```exact_hessian``` branch, so maybe this isn't worth bothering with right now.